packages/frontend/amp/server/document.tsx

```diff
@@ -24,14 +27,18 @@ export const document = ({
         // TODO: CacheProvider can be removed when we've moved over to using @emotion/core
         renderToString(<CacheProvider value={cache}>{body}</CacheProvider>),
     );
-
     return `<!doctype html>
 <html ⚡>
     <head>
     <meta charset="utf-8">
     <title>${title}</title>
     <link rel="canonical" href="self.html" />
     <meta name="viewport" content="width=device-width,minimum-scale=1">
+    
+    <script type="application/ld+json">
+        ${JSON.stringify(structuredData, null, 2)}
```

Comment:
Why does this need whitespace in the JSON?

Reply:
Good spot, removed.